rsclone/src/app/components/main-page/start-screen/start-screen.component.html

```diff
@@ -0,0 +1,149 @@
+                fill="#5E9641" />
+              <path id="Vector_14"
+                d="M411.519 144C416.019 169.124 398.02 180.936 388.458 195.56C371.584 221.809 396.708 249.183 424.644 246.745C454.268 244.308 464.58 219.934 453.893 193.31C445.268 171.936 426.331 150.562 411.519 144Z"
+                fill="#94C83D" />
+              <path id="Vector_15"
+                d="M411.519 144C412.082 147.562 412.269 150.75 412.082 153.75C416.582 161.437 423.331 174.936 426.706 190.31C426.706 190.873 426.894 191.248 427.081 191.81C428.581 199.685 429.144 207.935 428.206 216.184L401.02 181.311C400.832 181.123 400.645 180.748 400.27 180.561C398.208 182.998 396.145 185.436 394.27 187.873L425.581 227.996C423.706 233.621 420.894 238.871 416.957 244.12C416.394 244.87 416.019 245.808 416.019 246.558C418.832 246.933 421.832 247.12 424.831 246.745C425.394 246.745 426.144 246.558 426.706 246.558C438.706 228.746 439.831 209.06 436.643 191.623L447.518 180.186C445.83 177.373 444.143 174.749 442.268 172.124C442.081 172.311 441.706 172.499 441.518 172.686L434.018 180.561C430.269 167.249 424.456 156.187 420.332 149.062C417.332 147 414.332 145.312 411.519 144Z"
+                fill="#5E9641" />
+              <g id="head">
+                <path id="Ellipse12"
+                  d="M434.432 106.482C450.939 106.482 464.321 93.0998 464.321 76.5926C464.321 60.0853 450.939 46.7036 434.432 46.7036C417.925 46.7036 404.543 60.0853 404.543 76.5926C404.543 93.0998 417.925 106.482 434.432 106.482Z"
+                  fill="#FEB8B8" />
+                <path id="Path120"
+                  d="M438.909 63.1886L462.117 64.1186C465.037 64.1096 468.225 64.0066 470.449 62.1186C473.799 59.2696 473.238 53.8936 471.444 49.8776C466.444 38.6956 455.291 34.6896 443.044 35.0186C430.797 35.3476 417.964 39.4986 411.369 49.8186C404.774 60.1386 402.992 73.1706 405.476 85.1626C408.348 79.0648 412.769 73.8269 418.297 69.9718C423.826 66.1167 430.269 63.7792 436.984 63.1926L438.909 63.1886Z"
+                  fill="#2F2E41" />
+              </g>
+            </g>
+          </g>
+          <defs>
+            <clipPath id="clip0">
+              <rect width="930" height="550" fill="white" />
+            </clipPath>
+          </defs>
+        </svg>
+      </div>
+      <div class="start-screen__content">
+        <h1 class="content__header">green map</h1>
+        <p class="content__text">
+          Save NATURE - dispose of garbage correctly.
+        </p>
+      </div>
+      <app-theme-switcher></app-theme-switcher>
+    </div>
+  </div>
+  <div class="scroll-down">
+    <span class="material-icons">
```

Comment:
зачем класс с material-icons, если можно сразу материаловскую иконку взять?

Reply:
I just used it like a font icon. it will help me to change its size in adaptive very easily. I followed these instructions: http://google.github.io/material-design-icons/
upd: replaced _span_ tag with _i_ tag